tests/component_handlers/kafka_connect/test_connect_handler.py

```diff
@@ -276,36 +291,34 @@ def test_should_call_helm_upgrade_install_and_uninstall_when_clean_connector_wit
             ]
         )
 
-        helm_wrapper.assert_has_calls(
+        helm_wrapper_mock.assert_has_calls(
             [
-                mock.call.helm_uninstall(
+                mock.call.uninstall(
                     namespace="test-namespace",
                     release_name="test-connector-clean",
-                    suffix="-clean",
                     dry_run=True,
                 ),
-                mock.call.helm_upgrade_install(
-                    app="kafka-connect-resetter",
+                mock.call.upgrade_install(
+                    release_name="test-connector-clean",
+                    namespace="test-namespace",
+                    chart=f"{helm_repo_config.repository_name}/kafka-connect-resetter",
                     dry_run=True,
-                    helm_command_config=HelmCommandConfig(
-                        debug=False,
-                        force=False,
-                        timeout="5m0s",
+                    flags=HelmUpgradeInstallFlags(
+                        version="1.0.4",
```

Comment:
this is already set as the default value?

Reply:
Yeah it is the default. I moved it here:
https://github.com/bakdata/kpops/blob/7f6cf1d62efe2f511e5d0cdd44eeab24044413cb/kpops/cli/pipeline_config.py#L35